raisinbread/README.md

```diff
@@ -29,6 +29,16 @@ that is not the case, replace all `mysql` commands below by the necessary values
 `mysql -u user -p -h host database_name`*
 
 ##### Sourcing SQL
+The script `tools/raisinbread_refresh.php` includes functionality that drops all existing tables in the
+database and sources all of the RaisinBread data automatically. To run this script, navigate to the LORIS
+root directory and run the following command:
+
+```bash
+./tools/raisinbread_refresh.php
```

Comment:
The script can also be run using `make testdata` which is probably a little more clear.